docs/data.rst

```diff
@@ -68,6 +68,7 @@ A local user can be promoted to staff with the command::
 
     ./manage.py ihavepower username --password=password
 
+.. _add-wiki-doc:
```

Comment:
what is this?

Reply:
This is a reference for the link on line 110 in `docs/troubleshooting.rst`